src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocblocktaglocation/InputJavadocBlockTagLocationCorrect.java

```diff
@@ -0,0 +1,19 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocblocktaglocation;
+
+/**
+ * configuration: default
+ */
+public class InputJavadocBlockTagLocationCorrect {
+
+    /**
+     * Summary {@code inline tag}.
+     * Escaped &#64;version
+     * {@code @author, @deprecated}
+     * {@literal @see @serial @hidden}
+     * All @customTags should be ignored.
```

Comment:
please add `* '@return' literal in {@code @return} Javadoc tag.`

Reply:
done